nnpdf_data/nnpdf_data/new_commondata/ATLAS_2JET_7TEV_R06/uncertainties.yaml

```diff
@@ -900,27 +900,27 @@ definitions:
     treatment: MULT
     type: ATLASLUMI11
 bins:
-- art_sys_1: 8666.003969584071
-  art_sys_2: 16.560395175435904
-  art_sys_3: 10.04665923462056
-  art_sys_4: 0.12656551920924486
-  art_sys_5: 0.03248869152160565
-  art_sys_6: -0.005128755118771153
-  art_sys_7: 0.001538882777613067
-  art_sys_8: 7.956144721434874e-05
-  art_sys_9: 4.889240542760659e-05
-  art_sys_10: -1.6857000290806893e-05
-  art_sys_11: 3.4623806397831055e-06
-  art_sys_12: -9.73326695550526e-07
-  art_sys_13: -3.976035620790233e-07
-  art_sys_14: -1.9832179073202648e-08
-  art_sys_15: 5.042876412272401e-08
-  art_sys_16: -3.055464319753114e-08
-  art_sys_17: 8.299758199060853e-09
-  art_sys_18: -2.3278312338978964e-09
-  art_sys_19: 1.544485432184385e-09
-  art_sys_20: 5.301477069263894e-10
-  art_sys_21: -1.4029943620803375e-11
+- art_sys_1: -8666.003969584068
+  art_sys_2: 16.56039517543592
```

Comment:
Given that this many digits of precision are clearly wrong (because two runs of presumably the same code produce different results) I think the numbers should be truncated.

6-7 digits is probably more than the actual precision that we have.
The same is true for all other files where this happens.
I think it would be good (since this deals with utilities) to add an utility to truncate the outputs